.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## Proposed Changes
+
+Give a brief description of the changes in this PR. If this relates to a bug or a feature request, please link it below.
+
+<!-- Link with "Closes #ISSUE-NUMBER" -->
+
+## Types of Changes
+
+- [ ] Bug Fix
+- [ ] New Feature
+- [ ] Documentation Update
+
+## Checklist
+
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] I have added necessary documentation
+
+## Breaking Changes
+
+- [ ] I have not introduced breaking changes
```

Comment:
Is this something we generally want to disallow? Or is this something that could be done after discussions with us?

Reply:
The latter. I will add a note explaining it.